index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    
+    
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="topnav">
+
+     <h1 class="logo">Grace</h1>
+
+       <div class ='hambuger'>
+        <div class ="hambuger-bar"></div>
+        <div class ="hambuger-bar"></div>
+        <div class ="hambuger-bar"></div>
+
+       </div>
```

Comment:
- [ ] Kindly I recommend you to use `nav` tag for the hamburger button, The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.